cmd/minishift/cmd/start_preflight.go

```diff
@@ -107,7 +107,7 @@ func preflightChecksBeforeStartingHost() {
 		checkIsoUrl,
 		"Checking the ISO URL",
 		false, configCmd.WarnCheckIsoUrl.Name,
-		"See the 'Managing Minishift' topic for more information")
+		"See the 'Managing Minishift' topic (https://docs.openshift.org/latest/minishift/using/basic-usage.html) for more information")
```

Comment:
The name of the topic itself has also changed from 'Managing Minishift' to 'Basic Usage'. :)

Reply:
Done.